python/observation-publisher/publisher/config.py

```diff
@@ -0,0 +1,160 @@
+from dataclasses import dataclass
+from enum import Enum, unique
+from typing import List, Optional, Dict
+
+import arize.utils.types
+from hydra.core.config_store import ConfigStore
+
+
+@unique
+class ValueType(Enum):
+    FLOAT64 = 1
+    INT64 = 2
+    BOOLEAN = 3
+    STRING = 4
+
+
+@dataclass
+class RegressionPredictionOutput:
+    prediction_score_column: str
+
+    @property
+    def column_types(self) -> Dict[str, ValueType]:
+        return {self.prediction_score_column: ValueType.FLOAT64}
+
+
+@dataclass
+class BinaryClassificationPredictionOutput:
+    prediction_label_column: str
+    prediction_score_column: Optional[str] = None
+
+    @property
+    def column_types(self) -> Dict[str, ValueType]:
+        ct = {self.prediction_label_column: ValueType.STRING}
+        if self.prediction_score_column is not None:
+            ct[self.prediction_score_column] = ValueType.FLOAT64
+        return ct
+
+
+@dataclass
+class MulticlassClassificationPredictionOutput:
+    prediction_label_column: str
+    prediction_score_column: Optional[str] = None
```

Comment:
for multiclass the prediction_score_column must be in list type, because the scores are calculated for all possible class